tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import os
+import sys
+
+from fixtures.server_fixtures import *
+from fixtures.replay_fixtures import *
+from fixtures.db_fixtures import *
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..")))
```

Comment:
Why would you need this?

Reply:
if you run tests not from project root directory, you need to import main project, otherwise you won't have your project root in path.